apps/src/code-studio/assets/SpriteUpload.js

```diff
@@ -1,31 +1,48 @@
 import React from 'react';
```

Comment:
i didn't catch this before, but this file should use the .jsx extension instead of .js (for convention only -- there's no technical reason)